client/src/Components/NavigationBar.js

```diff
@@ -0,0 +1,185 @@
+import { NavLink } from "react-router-dom";
+import Avatar from "@material-ui/core/Avatar";
+import Menu from "@material-ui/core/Menu";
+import MenuItem from "@material-ui/core/MenuItem";
+import { DropzoneDialog } from "material-ui-dropzone";
+
+const useStyles = makeStyles((theme) => ({
+  appBar: {
+    background: "#FFFFFF",
+    boxShadow: "none",
+    flexDirection: "row",
+    alignItems: "center",
+    height: "85px",
+  },
+  toolBar: {
+    justifyContent: "space-between",
+    alignItems: "center",
+    flexGrow: 2,
+  },
+  buttonNotSelected: {
+    margin: theme.spacing(1),
+    color: "#666666",
+  },
+  selectedButton: {
+    margin: theme.spacing(1),
+    color: "#759DFD",
+  },
+  title: {
+    color: "black",
+  },
+  buttonText: {
+    fontSize: 14,
+    fontWeight: "bold",
+  },
+  buttonIcons: {
+    marginRight: "8px",
+  },
+  createButton: {
+    backgroundColor: "#759DFD",
+    marginRight: "40px",
+    width: "175px",
+    "&:hover": {
+      backgroundColor: "#759DFD",
+    },
+  },
+  createButtonText: {
+    color: "#FFFFFF",
+    fontWeight: "bold",
+    padding: "10px",
+  },
+  createButtonIcon: {
+    color: "#FFFFFF",
+  },
+  userIcon: {
+    width: "68px",
+    height: "68px",
+    flexShrink: 3,
+  },
+}));
+
+const NavigationBar = () => {
+  const initialStateDropFile = {
+    open: false,
+    files: [],
+  };
+
+  const [anchorEl, open] = React.useState(null);
+
+  const handleClickAvatarMenu = (event) => {
+    open(event.currentTarget);
+  };
+  const handleCloseAvatarMenu = () => {
+    open(null);
+  };
+
+  const [stateDropFile, setStateDropFile] = React.useState(
+    initialStateDropFile
+  );
+
+  const handleOpenDropFile = () => {
+    setStateDropFile({
+      ...stateDropFile,
+      open: true,
+    });
+  };
+
+  const handleCloseDropFile = () => {
+    setStateDropFile({
+      ...stateDropFile,
+      open: false,
+    });
+
+    handleCloseAvatarMenu();
+  };
+
+  const handleSaveDropFile = (files) => {
+    setStateDropFile({
+      ...stateDropFile,
+      files: files,
+      open: false,
+    });
+  };
+
+  const classes = useStyles();
+
+  return (
+    <AppBar position="static" className={classes.appBar}>
+      <Toolbar className={classes.toolBar}>
+        <Box>
+          <Button>
+            <Typography variant="title">
+              <img src="/images/logo.png" alt="bug" />
+            </Typography>
+          </Button>
+        </Box>
+        <Box>
+          <Button
+            className={classes.buttonNotSelected}
+            component={NavLink}
+            to="/"
+            activeClassName={classes.selectedButton}
+          >
+            <DashboardRoundedIcon className={classes.buttonIcons} />
+            <Typography className={classes.buttonText}>Dashboard</Typography>
+          </Button>
+          <Button
+            className={classes.buttonNotSelected}
+            component={NavLink}
+            to="/calendar"
```

Comment:
we may not not want the calendar button to navigate to a new page but to just render the calendar data instead by passing down a boolean key through the state. It can be discussed on the call with Aecio on the best approach towards this.